tests/core/fs/test_local_folder_fs.py

```diff
@@ -102,8 +102,7 @@ def test_file_create(local_folder_fs, alice):
     }
 
 
-@pytest.mark.parametrize("type", ["copy", "move"])
-def test_copy_folders_create_new_accesses(local_folder_fs, type):
+def test_copy_folders_create_new_accesses(local_folder_fs):
```

Comment:
Shouldn't we rename this `test_move_folder_create_new_accesses` ?

Reply:
Done!